src/app/components/dashboard/dashboardView.html

```diff
@@ -1,3 +1,289 @@
+        <div class="panel-heading">
+          <h3 class="text-center">Subscribed RSO Feed:</h3>
+        </div>
+        <div class="panel-body pre-scrollable">
+          <table class="table table-striped">
+            <thead>
+            <tr>
+              <th>Event</th>
+              <th>RSO</th>
+              <th>Date</th>
+              <th>University</th>
+              <th>Proximity</th>
+              <th>Details</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+              <td>Face Painting</td>
+              <td>SGA</td>
+              <td>10/20/2017</td>
+              <td>University of Central Florida</td>
+              <td>34.1 MI</td>
+              <td>
+                <div class="btn-group-vertical">
+                  <button class="btn btn-success">
+                    <i class="fa fa-pencil-square-o" aria-hidden="true"></i> Edit
+                  </button>
+                  <button class="btn btn-primary">
+                    <i class="fa fa-eye" aria-hidden="true"></i> View
+                  </button>
+                  <button class="btn btn-danger">
+                    <i class="fa fa-close" aria-hidden="true"></i> Leave
+                  </button>
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <td>Face Painting</td>
+              <td>SGA</td>
+              <td>10/20/2017</td>
+              <td>University of Central Florida</td>
+              <td>34.1 MI</td>
+              <td>
+                <div class="btn-group-vertical">
+                  <button class="btn btn-success">
+                    <i class="fa fa-pencil-square-o" aria-hidden="true"></i> Edit
+                  </button>
+                  <button class="btn btn-primary">
+                    <i class="fa fa-eye" aria-hidden="true"></i> View
+                  </button>
+                  <button class="btn btn-danger">
+                    <i class="fa fa-close" aria-hidden="true"></i> Leave
+                  </button>
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <td>Face Painting</td>
+              <td>SGA</td>
+              <td>10/20/2017</td>
+              <td>University of Central Florida</td>
+              <td>34.1 MI</td>
+              <td>
+                <div class="btn-group-vertical">
+                  <button class="btn btn-success">
+                    <i class="fa fa-pencil-square-o" aria-hidden="true"></i> Edit
+                  </button>
+                  <button class="btn btn-primary">
+                    <i class="fa fa-eye" aria-hidden="true"></i> View
+                  </button>
+                  <button class="btn btn-danger">
+                    <i class="fa fa-close" aria-hidden="true"></i> Leave
+                  </button>
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <td>Face Painting</td>
```

Comment:
Its all just html right now. Made the views as quickly as possible so Koha always had something to work on. Will add functionality this weekend.